### PR TITLE
add dmr info

### PR DESCRIPTION
**Pull request by samlachance**
Here is some basic info for DMR. I also went ahead and created a Digital Radio section that could be populated with Fusion and Dstar stuff.

### CODE REVIEW

File: _pages/channels/ham.md [hunk 1]
```diff
@@ -32,6 +32,33 @@ Discussions about Amateur Radio in the Chattanooga area.
...
+
+**General Info**
+
+[DMR - How it works](https://www.youtube.com/watch?v=5ZawJek5z9c) - Video intro to DMR
+
+[DMR-MARC](https://www.dmr-marc.net/cgi-bin/trbo-database/register.cgi) - Apply for your DMR ID
+
+**Good Radios**
+
+[TYT MD-380 (Amazon)](https://www.amazon.com/TYT-MD-380-Moto-TRBO-Radio/dp/B00X6FYWWS) - Great beginner DMR radio
```
Comment (agarzola) [inline _pages/channels/ham.md]: @StrangeWill Is devanooga signed up for an affiliate account? If so, let’s replace this link with an affiliate link. If not, let’s change the link to use the `smile.` sub-domain and add a note encouraging people to set devanooga as their preferred charity on the Smile website?

Comment (StrangeWill) [reply _pages/channels/ham.md]: Smile account is at: https://smile.amazon.com/ch/82-1082043

We don't currently have an affiliate link, I think using an affiliate link *may* be more messy from a tax perspective too.

Comment (samlachance) [reply _pages/channels/ham.md]: will do

**Comment by samlachance** [comment]
Done @agarzola

**Comment by agarzola** [comment]
Thanks for this, @samlachance! Should be live shortly.

**Pull request merged**